pkgs/games/xjump/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, gcc, autoconf, automake, xorg, ... }:
+
+stdenv.mkDerivation rec {
+  name = "xjump-${version}";
+  version = "2.9.3";
+  src = fetchFromGitHub {
+    owner = "hugomg";
+    repo = "xjump";
+    rev = "e7f20fb8c2c456bed70abb046c1a966462192b80";
+    sha256 = "0hq4739cvi5a47pxdc0wwkj2lmlqbf1xigq0v85qs5bq3ixmq2f7";
+  };
+  buildInputs = [ gcc autoconf automake xorg.libX11 xorg.libXt xorg.libXpm xorg.libXaw ];
+  preConfigure = ''
+    autoreconf --install
+  '';
+  installPhase = ''
```

Comment:
Doesn't look like a custom installPhase is needed, please remove.

Using default installation behavior also installs a manpage, icon, and .desktop file, which are all nice to include.

It also installs a "var/xjump/highscores" file which may be problematic, possibly needing to be removed in a postInstall or something-- although probably should check to ensure either way that it doesn't attempt to read/write that location regardless.

Reply:
Up to you if you want to fix the highscores situation, I think ignoring it for now is fine.

Reply:
Thanks for the review! I just fixed my mistakes (at least I hope), including the highscores, which are now configurable. I don't know what is the best way to install a system-wide highscores file, the behaviour now is to default to /var/xjump, and let the user create that directory and give it the appropriate permissions.